spek-runtime/src/commonMain/kotlin/org/spekframework/spek2/runtime/execution/Discovery.kt

```diff
@@ -0,0 +1,8 @@
+package org.spekframework.spek2.runtime.execution
+
+import org.spekframework.spek2.runtime.discovery.DiscoveryContext
+import org.spekframework.spek2.runtime.scope.Path
+import org.spekframework.spek2.runtime.scope.ScopeImpl
+
+data class DiscoveryRequest(val context: DiscoveryContext, val sourceDirs: List<String>, val paths: List<Path>)
```

Comment:
Would it make sense for `sourceDirs` to be on the JVM version of `DiscoveryContext` rather than on `DiscoveryRequest`?

Reply:
Yep, but I'll work on it on a separate PR to minimise the changes here.